client/web/src/site-admin/notifications/NotificationsNavbarItem.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useMemo } from 'react'
+
+import { mdiBellAlert, mdiBellBadge, mdiBellCheck, mdiCheckCircle, mdiInformation } from '@mdi/js'
+import classNames from 'classnames'
+import { groupBy, startCase } from 'lodash'
+
+import { renderMarkdown } from '@sourcegraph/common'
+import { Icon, Badge, Popover, PopoverTrigger, PopoverContent, Text, Button, PopoverTail } from '@sourcegraph/wildcard'
+
+import { AlertType } from '../../graphql-operations'
+
+import { useNotifications } from './hooks/useNotifications'
+
+import styles from './NotificationsNavbarItem.module.scss'
+
+const NotificationsGroup: React.FC<{ name: string; items: { message: string; type: AlertType }[] }> = ({
+    name,
+    items,
+}) => {
+    const hasError = items.find(item => item.type === AlertType.ERROR)
+    const hasWarning = items.find(item => item.type === AlertType.WARNING)
+    const hasInfo = items.find(item => item.type === AlertType.INFO)
+    return (
+        <div>
+            <div className="mb-2">
+                <Icon
+                    svgPath={hasError || hasWarning || hasInfo ? mdiInformation : mdiCheckCircle}
```

Comment:
```suggestion
                    svgPath={(hasError || hasWarning || hasInfo) ? mdiInformation : mdiCheckCircle}
```

What do you think about this? Makes it easier ti understand??

Reply:
It looks like we don't have any Eslint rules for this case, and I am okay with using any of the approaches. Updated.